packages/legend-query-builder/src/graph-manager/protocol/pure/v1/V1_QueryBuilder_PureGraphManagerExtension.ts

```diff
@@ -93,18 +95,23 @@ export class V1_QueryBuilder_PureGraphManagerExtension extends QueryBuilder_Pure
       v1Mappings.forEach((element) => {
         const mapping = mappings.find((e) => e.path === element.path);
         if (mapping) {
-          mapping.includes = element.includedMappings.map(
-            (mappingInclude) =>
-              new MappingInclude(
-                mapping,
-                context.resolveMapping(
-                  guaranteeNonEmptyString(
-                    mappingInclude.includedMapping,
-                    `Mapping include path is missing or empty`,
+          mapping.includes = element.includedMappings
+            .map((mappingInclude) => {
+              // Handle for dataspace include?
```

Comment:
i think we handle this already for dataspace include. if we have the logic in the first pass, the above `indexLightGraph` should probably handle that.